.github/workflows/build.yml

```diff
@@ -84,23 +84,20 @@ jobs:
     strategy:
       matrix:
         python-version: [3.7]
-        torch: [1.3.0, 1.5.0, 1.6.0, 1.7.0]
+        torch: [1.5.0+cu101, 1.6.0+cu101, 1.7.0+cu101, 1.8.0+cu101]
         include:
-          - torch: 1.3.0
-            torchvision: 0.4.1
-          - torch: 1.5.0
-            torchvision: 0.6.0
-          - torch: 1.6.0
-            torchvision: 0.7.0
-          - torch: 1.7.0
-            torchvision: 0.8.1
-          - torch: 1.7.0
-            torchvision: 0.8.1
-            python-version: 3.6
-          - torch: 1.7.0
-            torchvision: 0.8.1
-            python-version: 3.8
-
+          - torch: 1.5.0+cu101
+            torchvision: 0.6.0+cu101
+            mmcv: 1.5.0+cu101
+          - torch: 1.6.0+cu101
+            torchvision: 0.7.0+cu101
+            mmcv: 1.6.0+cu101
+          - torch: 1.7.0+cu101
+            torchvision: 0.8.1+cu101
+            mmcv: 1.7.0+cu101
+          - torch: 1.8.0+cu101
+            torchvision: 0.9.0+cu101
+            mmcv: 1.8.0+cu101
```

Comment:
Considering 3080/3090, it's necessary to check pt1.8.0 + cu111.

Reply:
We now only support cu101 arch

Reply:
mmcv support cu111? https://download.openmmlab.com/mmcv/dist/cu111/torch1.8.0/index.html

Reply:
I mean the CUDA environment